pkg/network/driver/common.go

```diff
@@ -227,57 +228,62 @@ func (h *NetworkUtilsHandler) IptablesAppendRule(proto iptables.Protocol, table,
 	return iptablesObject.Append(table, chain, rulespec...)
 }
 
-func (h *NetworkUtilsHandler) NftablesNewChain(proto iptables.Protocol, table, chain string) error {
-	// #nosec g204 no risk to use GetNFTIPString as  argument as it returns either "ipv6" or "ip" strings
-	output, err := exec.Command("nft", "add", "chain", h.GetNFTIPString(proto), table, chain).CombinedOutput()
+func (h *NetworkUtilsHandler) NftablesNewTable(proto iptables.Protocol, table string) error {
+	// #nosec g204 no risk to use GetNFTIPString as  argument as it returns either "ip6" or "ip" strings
+	output, err := exec.Command("nft", "add", "table", h.GetNFTIPString(proto), table).CombinedOutput()
 	if err != nil {
 		return fmt.Errorf("%s", string(output))
 	}
 
 	return nil
 }
 
-func (h *NetworkUtilsHandler) NftablesAppendRule(proto iptables.Protocol, table, chain string, rulespec ...string) error {
-	cmd := append([]string{"add", "rule", h.GetNFTIPString(proto), table, chain}, rulespec...)
-	// #nosec No risk for attacket injection. CMD variables are predefined strings
+func (h *NetworkUtilsHandler) NftablesNewChain(proto iptables.Protocol, table, chain string, chainspec ...string) error {
+	// #nosec g204 no risk to use GetNFTIPString as  argument as it returns either "ipv6" or "ip" strings
+	cmd := append([]string{"add", "chain", h.GetNFTIPString(proto), table, chain}, chainspec...)
 	output, err := exec.Command("nft", cmd...).CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("failed to apped new nfrule error %s", string(output))
+		return fmt.Errorf("%s", string(output))
 	}
 
 	return nil
 }
 
-func (h *NetworkUtilsHandler) GetNFTIPString(proto iptables.Protocol) string {
-	if proto == iptables.ProtocolIPv6 {
-		return "ip6"
+func (h *NetworkUtilsHandler) NftablesCreateBaseChains(proto iptables.Protocol) error {
+	if err := h.NftablesNewTable(proto, "nat"); err != nil {
+		return err
 	}
-	return "ip"
+	if err := h.NftablesNewChain(proto, "nat", "prerouting", "{ type nat hook prerouting priority -100; }"); err != nil {
+		return err
+	}
+	if err := h.NftablesNewChain(proto, "nat", "input", "{ type nat hook input priority 100; }"); err != nil {
+		return err
```

Comment:
Could you make sure that these errors are logged? 
Currently, when there's an error when loading the file, we log the error with
[`log.Log.V(5).Reason(err).Infof("failed to load nftable %s", fnName)`](https://github.com/kubevirt/kubevirt/pull/7628/commits/0d167942c0fdc69a17feae1aec93aea47060fa0b#diff-580ca5ffc6c25ac7ce3258b427568c362723ff5a74022f8d6bab61ff40f29737L266)

With this change, the returned error is thrown away at
https://github.com/kubevirt/kubevirt/pull/7628/commits/0d167942c0fdc69a17feae1aec93aea47060fa0b#diff-5cc59c701f44b3d4b9d3f0e79b5098fca04ac0c4f1218eb74c6d3cbe36da6649R225